PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/startup/HDInsightHelpDlg.java

```diff
@@ -20,11 +20,15 @@
 
 import com.microsoft.azuretools.core.Activator;
 import com.microsoft.azuretools.core.utils.Messages;
+import com.microsoft.azuretools.core.utils.PluginUtil;
 
 
 public class HDInsightHelpDlg extends Dialog {
 	private boolean isShowTips = true;
 	
+	private static String scalaPluginMarketplaceURL = "http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=421";
```

Comment:
Is the HDInsightHelpDlg only for Scala Plugin? If yes, it would be better to change the class name, otherwise, moving those hardcode URL out of class might be proper.